azext_edge/edge/_help.py

```diff
@@ -93,6 +93,8 @@ def load_iotops_help():
             - {COMPAT_MQTT_BROKER_APIS.as_str()}
             - {COMPAT_OPCUA_APIS.as_str()}
 
+            For more information on cluster requirements, please check https://aka.ms/iot-ops-cluster-requirements
```

Comment:
Note that this article is (currently) outdated but I believe in the maintainers to update it to reflect the latest release.

This is much easier than listing out everything the services support and updating it.